addons/l10n_sa/models/account_journal.py

```diff
@@ -0,0 +1,87 @@
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    @api.model
+    def _fill_missing_values(self, vals):
```

Comment:
A big part of this function is just a copy-paste from account. This is *very* bad practice. You need to find a way to override it that doesn't require to do that (possibly introducing a hook in account, if required). Why wasn't just an override enough here?

Reply:
I refactored the function to use super and only create 2 accounts